scripts/update-brew-formulae.sh

```diff
@@ -0,0 +1,25 @@
+#! /usr/bin/env bash
```

Comment:
Is this script used anywhere? :cant-find:

Reply:
Not yet, but it should find some use in #288. I wrote the script when I thought it might be used here and now and was told to 'keep it', so I thought I'd commit it anyway 🙈

Reply:
I guess we can leave a TODO for that issue in here then.